main.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+#include <cmath>
+#include <stdio.h>
+
+using namespace std;
+
+int main()
+{
+    setlocale(0,"");
+    float left, right;
+    char znak;
+    cin >>left>>znak>>right;
+    switch (znak)
+    {
+    case '+':
+        cout <<left+right;

```

Comment:
Result of arithmetic operations should be stored in appropriate variable and only then printed